thorlcr/msort/tsortl.cpp

```diff
@@ -315,11 +317,21 @@ ISocketRowWriter *ConnectMergeWrite(IThorRowInterfaces *rowif,ISocket *socket,si
         dst += read;
     }
     hdr.winrev();
+    if (hdr.hdrSig != transferStreamSignature)
```

Comment:
Do we need to winrev() hdrSig ?

Reply:
There seems no reason to have any of the winrev - both Thor manager and workers must be on the same platform/byte order.
It will be less confusing to remove.